### DISCUSSION TITLE
How to deal with streched fullscreen without X11

### DISCUSSION THREAD

**Discussion by RolandTi** [The-Wineskin-Project/WineskinServer#319] [Q&A] [answered]
So as I understand, X11 is no longer working with 64 bits wine.
If I start an old game who run in 1024x768 I'm stuck with an ugly stretched picture.
Is there any new way to keep aspect ratio ? Maybe using dgvoodoo 2 ?

Thanks ! :)

**Comment by Gcenx** [comment, marked as answer]
Yes you’d need to use some third-party utility, there’s cnc_ddraw & DxWnd verbs in winetricks

**Comment by RolandTi** [comment]
Oh yes, my game is working with DxWnd, thanks :)
